subscriptions/management/commands/add_missed_call_service_audio_notification_to_active_subscriptions.py

```diff
@@ -0,0 +1,32 @@
+from django.core.exceptions import ObjectDoesNotExist
```

Comment:
I would change the name of the file to something a little more generic, then we can always reuse for something else. Something like `add_prepend_next_to_subscriptions`.